src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java

```diff
@@ -1,23 +1,32 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- *
+ * <p>

```

Comment:
this tag should be removed from copyright

Reply:
That's right